Cargo.toml

```
@@ -68,7 +68,7 @@ default = ["std", "log", "libc", "errno", "signal", "init-fini-arrays", "program
 # If you're using nightly Rust, enable this feature to let origin use
 # nightly-only features, which include proper support for unwinding (if
 # enabled), better safety checks, and better optimizations.
-nightly = ["unwinding"]
+nightly = ["dep:unwinding"]
```

This was inaccurate. I was testing with `RUSTFLAGS='-C panic=abort'` which is the only way `Cargo.toml` can match on `panic = "unwinding"` condition. The use of `dep:` made no actual difference.

```suggestion
nightly = ["unwinding"]
```